server/chat-commands/room-settings.ts

```diff
@@ -1309,8 +1310,8 @@ export const roomSettings: SettingsHandler[] = [
 	(room, user) => {
 		const threshold = user.can('makeroom') ? Infinity :
 			user.can('modchatall', null, room) ? Config.groupsranking.indexOf(room.getAuth(user)) :
-			user.can('modchat', null, room) ? 1 :
-			null;
+				user.can('modchat', null, room) ? 1 :
+					null;
```

Comment:
I'm not a fan of enforcing increasing indentation on `?:` chains. Is there a way to turn this off?

Reply:
> Is there a way to turn this off?

Yes, [`flatTernaryExpressions` is toggle-able](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/indent.md#flatternaryexpressions).

Reply:
Fixed.